.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+ci:
```

Comment:
What does the `ci` block do here? I don't see it as a top level configuration here: https://pre-commit.com/#pre-commit-configyaml---top-level